TMBuilder/src/main/java/com/ebay/tool/thinmodelgen/gui/menu/TMBuilderMenu.java

```diff
@@ -325,7 +311,7 @@ private boolean validationSetCacheContainsSetWithName(String validationSetName)
   }
 
   private void doNewValidationSet() {
-    String name = JOptionPane.showInputDialog(MainWindow.getInstance(), "Enter new validation set name:");
+    String name = JOptionPane.showInputDialog(MainWindow.getInstance(), "Enter new validation set method name:");
```

Comment:
Since this is a dialog input could we add some additional clarifying text about the method name?

EG: Enter new validation set name.\nThis is the name that will be used when generating the validation methods.

Please confirm that the input dialog looks correct with this additional text.

Reply:
Yes we can and added as requested.